app/server/appsmith-server/src/main/java/com/appsmith/server/configurations/CommonConfig.java

```diff
@@ -66,6 +67,11 @@ public class CommonConfig {
 
     private List<String> allowedDomains;
 
+    private String mongoDBVersion;
+
+    @Value("${appsmith.productalert.minsupportedmongoversion:5}")
+    private String minSupportedMongoVersion;
```

Comment:
I don't think we need this to be configurable. Let's make it a constant instead please.

```suggestion
    private static final String MIN_SUPPORTED_MONGODB_VERSION = 5;
```

We'll need to rename the variable where it's used as well.